benchmark/intl_stream_benchmark.dart

```diff
@@ -0,0 +1,77 @@
+import 'dart:math';
```

Comment:
nit: add a copyright header to this new file

Reply:
Although - from this comment `I would remove the old method references from this file when merging, this is just for you to compare.` - I'm not sure if you plan to check this file in?